app/models/workers_scaling/experiment_statistics_factory.rb

```diff
@@ -0,0 +1,19 @@
+module WorkersScaling
+  class ExperimentStatisticsFactory
+    @cache = ActiveSupport::Cache::MemoryStore.new expires_in: 10.minutes
```

Comment:
Inna sprawa, że do cache zapisujemy klasy, żeby uniknąć ich ciągłego instancjonowania z tych samych parametrów. Jaki to będzie miało cel, jeżeli pod spodem i tak zapiszemy na dysk?